libs/s3/s3_iam_rest_rgw.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import asyncio
+from http import HTTPStatus
+import json
+import os
+import pathlib
+import sys
+import base64
+from hashlib import sha1
+import hmac
+from typing import Any, Dict, Optional, Tuple
+from aiohttp import ClientSession, ClientError
+from http import HTTPStatus
+import ssl
+from time import gmtime, strftime
+from typing import Any, Dict, Optional, Tuple
+from urllib.parse import urlencode
+from config import CMN_CFG
+
+ACCESS_KEY_ID = "sgiamadmin"
+SECRET_ACCESS_KEY = "ldapadmin"
```

Comment:
Can we move this parameter to secrets.json file or in config file with encryption in password manager

Reply:
Also make sure this is class variable so that it easier to change when calling from other libs

Reply:
Yes , can i go with Suggestion. cc @seagate-sarang-sawant

Reply:
Yes, by your suggestion can i go with that including on setup_lc.json

Reply:
updated , access key, secret key, and port as class variable